scripts/mkcloud

```diff
@@ -798,10 +802,16 @@ function prepareinstcrowbar()
     return $?
 }
 
+function scp_install_chef_suse_override()
+{
+    if [ -e "$install_chef_suse_override" ]; then
```

Comment:
maybe additionally look for `ìnstall-chef-suse.sh` in the working dir, as a fallback?